glimpse/helpers.py

```diff
@@ -1985,3 +1985,46 @@ def _parse_parallel(parallel):
     else:
         n = parallel
     return n
+
+def change_extenstion(infile,extension):
```

Comment:
This function isn't needed, since there already exists the more general `helpers.strip_path` (https://github.com/ezwelty/glimpse/blob/master/glimpse/helpers.py#L111): e.g. `helpers.strip_path(path) + '.ext'`